app.hypermedia.testing.dsl/src/main/java/app/hypermedia/testing/dsl/Hydra.xtext

```diff
@@ -35,4 +49,9 @@ InvocationBlock:
     {InvocationBlock} 'Invoke' '{'
         (children+=ResponseStep)*
     '}'
-;
\ No newline at end of file
+;
+
+terminal URI:
+    '<' ( '\\' . /* 'b'|'t'|'n'|'f'|'r'|'u'|'"'|"'"|'\\' */ | !('\\'|'>') )* '>'
+;
+terminal QNAME: ID ':' ID;
```

Comment:
What would it take to turn this into a parser rule and have the prefix cross reference `PrefixDeclaration`?